tests/integration/test_init.py

```diff
@@ -250,7 +265,9 @@ def test_init_search_for_specfile_root(upstream_without_config_not_bare):
             assert "\nspecfile_path: file.spec\n" in f.read()
 
 
-def test_init_search_for_specfile_recursive(upstream_without_config_not_bare):
+def test_init_search_for_specfile_recursive(
+    upstream_without_config_not_bare,
+):
```

Comment:
Weird formatting changes that shouldn't have been included, but pre-commit is a thing I guess.